src/module.ts

```diff
@@ -63,26 +63,12 @@ export default defineNuxtModule<ModuleOptions>({
     })
 
     if (options.highlight) {
-      // Enable unwasm for shiki
-      nuxt.hook('ready', () => {
-        const nitro = useNitro()
-        const addWasmSupport = (_nitro: typeof nitro) => {
-          if (nitro.options.experimental?.wasm) { return }
-          _nitro.options.externals = _nitro.options.externals || {}
-          _nitro.options.externals.inline = _nitro.options.externals.inline || []
-          _nitro.options.externals.inline.push(id => id.endsWith('.wasm'))
-          _nitro.hooks.hook('rollup:before', async (_, rollupConfig) => {
-            const { rollup: unwasm } = await import('unwasm/plugin')
-            rollupConfig.plugins = rollupConfig.plugins || []
-              ; (rollupConfig.plugins as any[]).push(unwasm({
-                ..._nitro.options.wasm as any,
-              }))
-          })
-        }
-        addWasmSupport(nitro)
-        nitro.hooks.hook('prerender:init', (prerenderer) => {
-          addWasmSupport(prerenderer)
-        })
+      // Install nuxt-shiki
+      // @see https://github.com/pi0/nuxt-shiki#nuxt-shiki
+      installModule('nuxt-shiki', {
+        bundledThemes: options.highlight.themes,
```

Comment:
Can you test if it still extends `shiki:` config key options? (normally kit should merge just double checking)

Reply:
I can test this, the only reason I passed it in was because it was defined on the mdc config already.

Reply:
Just tested and it still can extend from `shiki:`